SPECS/selinux-policy/selinux-policy.spec

```diff
@@ -0,0 +1,14032 @@
+# upstream does not currently support a mariner distro
+%define distro redhat
```

Comment:
Could you create a bug for this to be fixed so that we don't lose track of it in the future?

Reply:
Sent an e-mail on this.  It's really part of a much larger chunk of work to generally tune the policy to Mariner, rather than just changing things for this one variable.